### PR TITLE
LedgerHandle: add counters to track ledger->bookie counts

### PR DESCRIPTION
**Pull request by athanatos**
We can use this to detect distribution issues.

(@bug W-4188471@)
Signed-off-by: Venkateswararao Jujjuri (JV) <vjujjuri@salesforce.com>
[Ported to master, cleaned up a bit, added test]
Signed-off-by: Samuel Just <sjust@salesforce.com>

### CODE REVIEW

**Comment by athanatos** [comment]
@jvrao I refactored it slightly to reduce duplication, you might want to glance over it.

File: bookkeeper-server/src/main/java/org/apache/bookkeeper/client/LedgerCreateOp.java [hunk 1]
```diff
@@ -209,6 +215,15 @@ public void operationComplete(int rc, Void result) {
             createComplete(BKException.Code.IncorrectParameterException, null);
             return;
         }
+
+        List<BookieSocketAddress> curEns = lh.getLedgerMetadata().getEnsemble(0L);
+        LOG.info("Ensemble: {} for ledger: {}", curEns, lh.getId());
```
Comment (eolivelli) [inline bookkeeper-server/src/main/java/org/apache/bookkeeper/client/LedgerCreateOp.java]: Change log level?

Comment (athanatos) [reply bookkeeper-server/src/main/java/org/apache/bookkeeper/client/LedgerCreateOp.java]: We actually like it at info.  @jvrao thoughts?

Comment (athanatos) [reply bookkeeper-server/src/main/java/org/apache/bookkeeper/client/LedgerCreateOp.java]: @eolivelli Would it be possible to leave it at info, or are there use cases you are worried about where the volume of ledger creation would be too high?

Comment (eolivelli) [reply bookkeeper-server/src/main/java/org/apache/bookkeeper/client/LedgerCreateOp.java]: Ok

**Pull request closed without merging**